Update postgres connection parameters for TFE (#1067)
hashicorp/terraform-website: content/source/docs/enterprise/before-installing/postgres-requirements.html.md

When providing optional extra keyword parameters for the database connection,
note an additional restriction on the `sslmode` parameter is that only the
`require`, `verify-full`, `verify-ca`, and `disable` values are allowed. The default value of `sslmode` is set to `require` with `external services` installation or `disable` with `demo` installation.

> -> **Note:** See PostgreSQL library for available extra parameters in relation to [sslmode](https://www.postgresql.org/docs/9.6/libpq-ssl.html). Terraform Enterprise provides certificates file at this location `/tmp/cust-ca-certificates.crt` in which required by `verify-full`, and `verify-ca` mode. Additional certificates can be added via [CA Custom Bundle](../install/installer.html#certificate-authority-ca-bundle) setting.

Reviewer: ```suggestion
-> **Note:** See the PostgreSQL library documentation for more about [extra parameters related to sslmode](https://www.postgresql.org/docs/9.6/libpq-ssl.html). Terraform Enterprise provides a certificates file at `/tmp/cust-ca-certificates.crt`, which is required by the `verify-full` and `verify-ca` modes. Additional certificates can be added via the [CA Custom Bundle](../install/installer.html#certificate-authority-ca-bundle) setting.
```

I might want to adjust that more, though — can you clarify what you mean about the certificates file? I read it in two different ways: 

- TFE populates this file automatically; if you're using one of the verify modes, you need to set an extra parameter to make sure Postgres knows where to find that file. 
- FYI: TFE automatically populates this file AND automatically supplies it to Postgres.
